src/core/PublicAPI.Unshipped.txt

```diff
@@ -464,6 +464,7 @@ Vezel.Cathode.Text.Control.ControlBuilder.SetWorkingDirectory(System.Uri! uri) -
 Vezel.Cathode.Text.Control.ControlBuilder.SoftReset() -> Vezel.Cathode.Text.Control.ControlBuilder!
 Vezel.Cathode.Text.Control.ControlBuilder.Space() -> Vezel.Cathode.Text.Control.ControlBuilder!
 Vezel.Cathode.Text.Control.ControlBuilder.Span.get -> System.ReadOnlySpan<char>
+Vezel.Cathode.Text.Control.ControlBuilder.Memory.get -> System.ReadOnlyMemory<char>
```

Comment:
This list needs to be sorted. If you use VS or VS Code, there should be a code action on the `Memory` property to add it to this file in the correct place.

Reply:
Ah I didn't realize there was a fixup. Addressed and pushed.